### PR TITLE
Implement Use Affected Rows

### PR DESCRIPTION
**Pull request by caleblloyd**
See #104 for more information

### CODE REVIEW

File: src/MySqlConnector/Serialization/MySqlSession.cs [hunk 1]
```diff
@@ -85,73 +85,9 @@ public MySqlSession(ConnectionPool pool, int poolGeneration)
...
-#endif
-					m_conversation = null;
-					m_hostname = "";
-					m_payloadHandler = null;
-					m_state = State.Failed;
-					m_tcpClient = null;
-					throw new MySqlException("SSL Authentication Error", ex);
-				}
-			}
+				await InitSslAsync(cs, ioBehavior, cancellationToken);
```
Comment (bgrainger) [inline src/MySqlConnector/Serialization/MySqlSession.cs]: Use `.ConfigureAwait(false)`.

File: docs/content/connection-options.md [hunk 2]
```diff
@@ -150,4 +150,9 @@ These are the other options that MySqlConnector supports.  They are set to sensi
     <td>false</td>
     <td>When set to false or no (strongly recommended), security-sensitive information, such as the password, is not returned as part of the connection if the connection is open or has ever been in an open state. Resetting the connection string resets all connection string values, including the password. Recognized values are true, false, yes, and no.</td>
   </tr>
+  <tr>
+    <td>Use Affected Rows, UseAffectedRows</td>
+    <td>false</td>
```
Comment (bgrainger) [inline docs/content/connection-options.md]: Should be `true`.

File: src/MySqlConnector/Serialization/MySqlSession.cs (hunk 1 above)
Comment (caleblloyd) [inline src/MySqlConnector/Serialization/MySqlSession.cs]: gets me every time

**Comment by caleblloyd** [comment]
Code review changes complete

**Pull request merged**